app/code/Magento/Catalog/Model/Product/Image/Process/Queue.php

```diff
@@ -0,0 +1,323 @@
+<?php
```

Comment:
Magento does not encourage inheritance-based APIs, therefore `protected` class properties should be made `private` when it's possible.